signal.go

```diff
@@ -49,18 +38,16 @@ func TerminateAll(ctx context.Context, procs []Process) error {
 	}
 
 	for _, proc := range procs {
-		proc.Wait(ctx)
+		_, _ = proc.Wait(ctx)
```

Comment:
why not catch this error?

Reply:
The error has never been checked on this call to `Wait()` because it will only return nil error if `(*Cmd) Wait()` runs and exits with zero exit status. This should happen on Unix systems but not on Windows. I guess I could write a platform-specific error check here for non-Windows systems.